### ISSUE TITLE
Add user guide for direct usage with containerit, o2r-meta, and erc-checker

### ISSUE THREAD

**Issue by nuest** [o2r-project/erc-spec#52]
Something along the lines of _"this is how you can check **before uploading** if we can create a runtime manifest (using `containerit`) and what kind of metadata we will be able to extract (with `o2r-meta extract`)"._

**Comment by ghost** [comment]
> what kind of metadata we will be able to extract (with o2r-meta extract)

Does this fit in the template section (e.g. geo template), where yaml headers are prepared that can be filled out in a workspace's rmd?

**Comment by nuest** [comment]
Yes, it's related to the templates and the content should be linked.
